docs/website/docs/dlt-ecosystem/verified-sources/sql_database.md

```diff
@@ -255,6 +255,44 @@ def sql_table(
 
 `write_disposition`: Can be "merge", "replace", or "append".
 
+## Incremental Configuration
```

Comment:
I would suggest to rename to Incremental Loading, because it may seem familiar to user from other parts of dlt docs

Reply:
done ))